CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a
 Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to
 the [PEP 440 version scheme](https://peps.python.org/pep-0440/#version-scheme).
 
+## [0.4.0] - 2024-07-18
+### Updated
```

Comment:
Updated is not a recognized heading. See https://keepachangelog.com/en/1.0.0/

![image](https://github.com/user-attachments/assets/a0a906e0-2ed9-4b1b-9f87-600fbc47ca48)


Use "Changed".

Reply:
The change log should let us know that the component was basically unusable due to a dependency conflict and this release fixes that.

Reply:
Done